src/components/index.js

```diff
@@ -71,3 +71,5 @@ export { default as RichContenteditable } from './RichContenteditable/index.js'
 export { default as SettingsSection } from './SettingsSection/index.js'
 export { default as UserBubble } from './UserBubble/index.js'
 export { default as Button } from './Button/index.js'
+export { default as InputField } from './InputField/index.js'
```

Comment:
I thought you didn't want to expose this?

Reply:
you're right